forge-gui/res/cardsfolder/upcoming/the_master_mesmerist.txt

```diff
@@ -0,0 +1,11 @@
+Name:The Master, Mesmerist
+ManaCost:2 U B
+Types:Legendary Creature Time Lord Rogue
+PT:3/3
+A:AB$ Pump | Cost$ T | ValidTgts$ Creature.OppCtrl+powerLEX | TgtPrompt$ Select target creature an opponent controls with power less than or equal to NICKNAME's power | KW$ Skulk | SubAbility$ DBGoad | StackDescription$ {c:Targeted} gains skulk until end of turn. Goad it. | SpellDescription$ Target creature an opponent controls with power less than or equal to NICKNAME's power gains skulk until end of turn. Goad it. (A creature with skulk can't be blocked by creatures with greater power.)

```

Comment:
Does this target prompt work with NICKNAME?

Reply:
@Glorax
probably last one to check
after only the bad delete commit of _Clockwork Droid_ needs to be sorted out